resources/index.html

```diff
@@ -5,6 +5,7 @@
 	<meta charset="UTF-8" />
 	<meta name="robots" content="noindex,nofollow" />
 	<meta name="googlebot" content="noindex,nofollow" />
+	<link rel="shortcut icon" type="image/x-icon" href="/bahmni/favicon.ico">
```

Comment:
The shortcut link type is often seen before icon, but this link type is non-conforming, ignored and web authors must not use it anymore. refer: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel